src/shimoku/api/resources/action.py

```diff
@@ -0,0 +1,130 @@
+from typing import Optional, TYPE_CHECKING, TypedDict
+
+import asyncio
+
+from shimoku.api.base_resource import Resource
+
+if TYPE_CHECKING:
+    from shimoku.api.resources.universe import Universe
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+MAX_CODE_FRAGMENT_SIZE = 8000
```

Comment:
Is this arbitrary?

Reply:
For now yes, i think it is the maximum size that back end decided to accept, but i am not sure, either way, we should make a deeper analysis to find the optimal fragment size